isis/src/base/objs/LidarData/LidarData.cpp

```diff
@@ -54,10 +53,20 @@ namespace Isis {
    *
    * @return @b QList<QSharedPointer<LidarControlPoint>> Returns list of Lidar control points.
```

Comment:
should there be a better description here, e.g. explaining the bool argument and the sorting stuff?